src/app/lib/config/services/ukchina.js

```diff
@@ -185,7 +185,8 @@ export const service = {
         },
       },
       media: {
-        noJs: '你的器材不支持播放多媒体材料',
+        noJS: '你的器材不支持播放多媒体材料',
```

Comment:
I think this is supposed to be noJs (lowercase s)

Reply:
Yes it is. I fixed it and then encountered a merge conflict. Must have lost the fix.